### ISSUE TITLE
Best way to have more support for termURL in tabular files ?

### ISSUE THREAD

**Issue by neurorepro** [bids-standard/bids-specification#1497]
Having `termURL` fields as described [here](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html) is great, but it seems limited to the column names and not e..g. to the categorical `Levels`. 

For example, imagining we have the `group` column in our tabular data for participants and want to indicate different subgroups of Alzheimer's subjects (AD, MCI, NC)  with a term URL for each, how we would do that ? 

```
  "group": {
      "Description": "Alzheimer's subgroups",
      "Levels": {
          "AD": "Alzheimer's disease", # want to indicate `termURL` http://terminology/AD.html here
          "MCI": "Mild cognitive impairment" # want to indicate `termURL` http://terminology/MCI.html here
          ...
      }
  }
```

Is the only solution for now to replace the level description with the URL itself ? Are there any developments planned for a more formal description ? A first step could be nested description/termURL for levels as in:
```
  "group": {
      "Description": "Alzheimer's subgroups",
      "Levels": {
          "AD": {
            "Description": "Alzheimer's disease", 
            "termURL": "https://terminology/AD.html"
           }
          "MCI": {
            "Description": "Mild cognitive impairment",
            "termURL": "https://terminologyMCI.html"
          }
          ...
      }
  }
```

Note that it could  be useful to also have this feature applied to Units as in:

    "volume": {
        "Description": "Volume of X",
        "Units": {
          "Description": "millilitre",
          "termURL": "https://terminology/mL.html"
        }
    }

**Comment by effigies** [comment]
Yes, the insufficiency of our column descriptions has come up recently. I like the idea of expanding plain-text descriptions to description fields that allow richer annotations. Thus `"AD": "Alzheimer's disease"` becomes shorthand for `"AD": {"Description": "Alzheimer's disease"}` and tools can easily perform this conversion and enable addition of new fields.

@surchs I would be interested in your thoughts here.

There's a general problem in BIDS, which is that we don't want to make it difficult to start structuring your data, but it would be good to find ways to enable "promoting" first-pass annotations to rich annotations. I would also like to do this with dataset authors...

**Comment by Remi-Gau** [comment]
potentially related to this: https://github.com/bids-standard/bids-specification/issues/615
